default.nix

```diff
@@ -12,6 +12,7 @@ with import <nixpkgs> {}; {
       dfu-util
       cmake
       libusb
+      cacert
     ];
     SSL_CERT_FILE="/etc/ssl/certs/ca-bundle.crt";
```

Comment:
Since setting this environment variable is done by the cacert setup hook and is overwritten anyway by the specific path into the nix store, you could just removed this for clarity.

Reply:
Done, thanks!